src/themes/tokens.css

```
@@ -0,0 +1,254 @@
+@theme {
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)